packages/opentelemetry-shim-opentracing/test/Shim.test.ts

```diff
@@ -190,7 +190,20 @@ describe('OpenTracing Shim', () => {
 
     it('can set and retrieve baggage', () => {
```

Comment:
can you add here one more test where you set baggage item again with different value for the same key and make sure the value has been set correctly ?

Reply:
Added.